### ISSUE TITLE
Chapter 2: Error with the check_image function

### ISSUE THREAD

**Issue by GomorrhaDev** [falloutdurham/beginners-pytorch-deep-learning#79]
![image](https://user-images.githubusercontent.com/108148171/202773982-338bea47-fbea-4345-bef0-b69ae731b44d.png)

**Comment by MarcusFra** [comment]
Hi @FoxxUI!

Could you please share the versions you had installed?
I just ran it on Google Colab and it works fine.
Versions installed:
```
torch: 1.12.1+cu113
torchvision: 0.13.1+cu113
PIL: 7.1.2
```

I made a few changes in order to run it on Colab, see here: https://github.com/MarcusFra/beginners-pytorch-deep-learning/blob/google_colab_ch2/chapter2/Chapter%202_with_Colab.ipynb (you need to adjust the paths to train, val and test set)
Does that help you?
